locales/uk/home.json

```diff
@@ -0,0 +1,145 @@
+{
+  "landingPage": {
+    "menu": {
```

Comment:
why there are Russian strings in the UK locale?

Reply:
This is Ukraine file
Not en